src/integrations/feed/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './types';
+export {
+  FeedPreferencesConfigGenerator,
+  SimpleFeedConfigGenerator,
+} from './configs';
+export { FeedClient, CachedFeedClient } from './clients';
+export {
+  FeedGenerator,
+  feedGenerators,
+  versionToFeedGenerator,
+  cachedFeedClient,
+} from './generators';
```

Comment:
We use index export in webapp lately, we should continue doing it for api too, this is a good start.

Reply:
I love index exports and try to use them where I can. It creates a design of small libraries within the project. Make a lot of sense to me